doc/book/zend.authentication.adapter.dbtable.md

```diff
@@ -0,0 +1,304 @@
+# Database Table Authentication
+
+> **note**
+>
+> `Zend\Authentication\Adapter\DbTable` has been deprecated, as its responsibilities have been
+splitted off into `Zend\Authentication\Adapter\DbTable\CallbackCheck` and
+`Zend\Authentication\Adapter\DbTable\CredentialTreatmentAdapter`. Use
+`Zend\Authentication\Adapter\DbTable\CredentialTreatmentAdapter` instead of
+`Zend\Authentication\Adapter\DbTable`.
+
+## Introduction
+
+`Zend\Authentication\Adapter\DbTable` provides the ability to authenticate against credentials
+stored in a database table. Because `Zend\Authentication\Adapter\DbTable` requires an instance of
+`Zend\Db\Adapter\Adapter` to be passed to its constructor, each instance is bound to a particular
+database connection. Other configuration options may be set through the constructor and through
+instance methods, one for each option.
+
+The available configuration options include:
+
+-   **tableName**: This is the name of the database table that contains the authentication
```

Comment:
I'm guessing the extra space between the `-` and the text was due to pandoc. We may need to write a post-processor to fix stuff like that.